driver/annotations.md

```diff
@@ -2,7 +2,7 @@
 
 # Intro
 
-Once you have written the [AST parser](sdk.md) the next step is to write
+Once you have written the [native code to AST parser](sdk.md) the next step is to write
```

Comment:
I think this is clearer (it's the `intro.md` file), this is not the code API where we've to be succinct. Another option: *code to AST parser* (without the *native*).